sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -1,5 +1,3 @@
-# Release History
```

Comment:
Release history heading should remain.

Reply:
Thanks. Fixed in this PR https://github.com/Azure/azure-sdk-for-net/pull/9168/files